packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -92,6 +115,9 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 		'level-2': level === 2 && ! completed,
 		'level-1': level === 1 && ! completed,
 	} );
+	if ( showActionButton === undefined ) {
+		showActionButton = expandable;
+	}
```

Comment:
Set `showActionButton` to `expandable` by default, this can be overwritten.
The action button only shows up when something is expanded, this is why we default `showActionButton` to the `expanded` value.